core/storage/base_model/gae_models.py

```diff
@@ -54,10 +54,25 @@
 )
 
 EXPORT_POLICY = utils.create_enum(  # pylint: disable=invalid-name
+    # Indicates that a model's field is to be exported.
     'EXPORTED',
+    # Indicates that the field is exported as the Takeout key.
```

Comment:
...that the value of the field...

...as the key in the Takeout dict.

Reply:
done